### PR TITLE
Isomap with more dimensions

### PR DESCRIPTION
**Pull request by KT12**
Re-ran the same analysis with Isomap dimensionality reduction, but with a larger number of components retained.  Results were similarly average.

### CODE REVIEW

File: algorithms/scripts/SGDClassifier-Isomap-KT12.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 # # Create a logistic regression model to predict TP53 mutation from gene expression data in TCGA using Isomap as a dimensionality reduction technique
 
-# In[1]:
+# In[27]:
```
Comment (dhimmel) [inline algorithms/scripts/SGDClassifier-Isomap-KT12.py]: Ah the diff would be way nicer with a freshly run notebook (so cells start at 1). Is is feasible to restart and run all cells, so this part of the diff goes away?

**Review by dhimmel — COMMENTED** [review]
So this PR increases in the number of components from 64 to 256 and adds parallel computation. Did performance change at all?

File: algorithms/scripts/SGDClassifier-Isomap-KT12.py (hunk 1 above)
Comment (KT12) [reply algorithms/scripts/SGDClassifier-Isomap-KT12.py]: Sorry about that - must have made another change and re-run the notebook.

Test AUROC went from ~63% to ~75%.

Comment (dhimmel) [reply algorithms/scripts/SGDClassifier-Isomap-KT12.py]: > Test AUROC went from ~63% to ~75%.

Wow! So it was in part an issue of too few features.

**Comment by KT12** [comment]
Unfortunately, there is no random seed to set for Isomap dimensionality reduction.  This time the AUROC was ~ 70%.

**Comment by dhimmel** [comment]
> Unfortunately, there is no random seed to set for Isomap dimensionality reduction.

Wow, that is a major drawback.

PR looks good.

**Review by dhimmel — APPROVED** [review]

**Pull request merged**